history.json

```diff
@@ -0,0 +1,199 @@
+    "flags": "b",
+    "piece": "p",
+    "originalString": "d4"
+  },
+  {
+    "color": "b",
+    "captured": "p",
+    "from": "e5",
+    "to": "d4",
+    "flags": "c",
+    "piece": "p",
+    "originalString": "exd4"
+  },
+  {
+    "color": "w",
+    "captured": "p",
+    "from": "d1",
+    "to": "d4",
+    "flags": "c",
+    "piece": "q",
+    "originalString": "Qxd4"
+  },
+  {
+    "color": "b",
+    "from": "b8",
+    "to": "c6",
+    "flags": "n",
+    "piece": "n",
+    "originalString": "Nc6"
+  },
+  {
+    "color": "w",
+    "from": "d4",
+    "to": "a4",
+    "flags": "n",
+    "piece": "q",
+    "originalString": "Qa4"
+  },
+  {
+    "color": "b",
+    "from": "g8",
+    "to": "f6",
+    "flags": "n",
+    "piece": "n",
+    "originalString": "Nf6"
+  },
+  {
+    "color": "w",
+    "from": "b1",
+    "to": "c3",
+    "flags": "n",
+    "piece": "n",
+    "originalString": "Nc3"
+  },
+  {
+    "color": "b",
+    "from": "d7",
+    "to": "d5",
+    "flags": "b",
+    "piece": "p",
+    "originalString": "d5"
+  },
+  {
+    "color": "w",
+    "captured": "p",
+    "from": "e4",
+    "to": "d5",
+    "flags": "c",
+    "piece": "p",
+    "originalString": "exd5"
+  },
+  {
+    "color": "b",
+    "from": "d8",
+    "to": "e7",
+    "flags": "n",
+    "piece": "q",
+    "originalString": "Qe7+"
+  },
+  {
+    "color": "w",
+    "from": "e1",
+    "to": "d1",
+    "flags": "n",
+    "piece": "k",
+    "originalString": "Kd1"
+  },
+  {
+    "color": "b",
+    "from": "c8",
+    "to": "g4",
+    "flags": "n",
+    "piece": "b",
+    "originalString": "Bg4+"
+  },
+  {
+    "color": "w",
+    "from": "d1",
+    "to": "d2",
+    "flags": "n",
+    "piece": "k",
+    "originalString": "Kd2"
+  },
+  {
+    "color": "b",
+    "captured": "p",
+    "from": "f6",
+    "to": "d5",
+    "flags": "c",
+    "piece": "n",
+    "originalString": "Nxd5"
+  },
+  {
+    "color": "w",
+    "from": "c3",
+    "to": "b5",
+    "flags": "n",
+    "piece": "n",
+    "originalString": "Nb5"
+  },
+  {
+    "color": "b",
+    "from": "c6",
+    "to": "b4",
+    "flags": "n",
+    "piece": "n",
+    "originalString": "Ncb4"
+  },
+  {
+    "color": "w",
+    "from": "c2",
+    "to": "c3",
+    "flags": "n",
+    "piece": "p",
+    "originalString": "c3"
+  },
+  {
+    "color": "b",
+    "from": "e8",
+    "to": "c8",
+    "flags": "q",
+    "piece": "k",
+    "originalString": "O-O-O"
+  },
+  {
+    "color": "w",
+    "from": "f2",
+    "to": "f3",
+    "flags": "n",
+    "piece": "p",
+    "originalString": "f3"
+  },
+  {
+    "color": "b",
+    "from": "e7",
+    "to": "e3",
+    "flags": "n",
+    "piece": "q",
+    "originalString": "Qe3+"
+  },
+  {
+    "color": "w",
+    "from": "d2",
+    "to": "d1",
+    "flags": "n",
+    "piece": "k",
+    "originalString": "Kd1"
+  },
+  {
+    "color": "b",
+    "captured": "p",
+    "from": "d5",
+    "to": "c3",
+    "flags": "c",
+    "piece": "n",
+    "originalString": "Nxc3#"
+  }
+]
```

Comment:
why this file?

Reply:
i dont remember adding this in - maybe from one of your PRs?